source/_components/media_player.firetv.markdown

```diff
@@ -117,3 +117,21 @@ If you receive the error message `Issue: Error while setting up platform firetv`
 3. Home Assistant does not have the appropriate permissions for the `adbkey` file and so it is not able to use it.  Once you fix the permissions, the component should work.
 
 4. You are already connected to the Fire TV via ADB from another device.  Only one device can be connected, so disconnect the other device, restart the Fire TV (for good measure), and then restart Home Assistant.  
+
+
+## {% linkable_title Service %}
```

Comment:
This is the old style of documenting services. Please use the new style. Look at [persistent_notification](https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/source/_components/persistent_notification.markdown) for an example

Reply:
Fixed